### PR TITLE
2d pointof the Home Work

### PR DESCRIPTION
**Pull request by SerhiiRiabko**
2d point of the homework

### CODE REVIEW

File: lesson_2/HomeWork_2.py [hunk 1]
```diff
@@ -1 +1,4 @@
-test
\ No newline at end of file
+Eleks = ['Rudolf Emir', 'Octav Marango', 'Dominik Toreto', 'Jeccy Tailor', 'Whitny Hyston']
+Toshiba = ['Akida Yamato', 'Yoshida Takad', 'Winhei Huan', 'Triseto Ariso', 'Ritero Naruto', 'Shibao Tikoo']
+Toshiba = Toshiba + Eleks
```
Comment (bohdandovbsyh) [inline lesson_2/HomeWork_2.py:3]: As mentioned in the lesson, try not to use + for adding lists. It has a particular function for this.  

Also after your manipulations Eleks should not have employees

File: lesson_2/home_work2_4.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+omnivore = ["Karl", "Klara", "Rianna", "Faust"]
+vegetarian = ["Walter", "Grubber", "Rikhter", "Rebecca"]
+
+eat_veg_herb = omnivore + vegetarian
```
Comment (bohdandovbsyh) [inline lesson_2/home_work2_4.py:4]: change + to smth else

File: lesson_2/home_work2_5.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+places_vip = {1: "Criss", 2: "Den", 3: "Aicek", 4: "Riddik" }
```
Comment (bohdandovbsyh) [inline lesson_2/home_work2_5.py:1]: VIP places are fixed. So it can no be mutable type

File: lesson_2/home_work2_6.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+group_of_people = ["John", "Rebecca", "Alessandro", "Melena", "John"]
+uniq_names = []
+for names in group_of_people:
+    if names not in uniq_names:
+       uniq_names.append(names)
+print(uniq_names)
```
Comment (bohdandovbsyh) [inline lesson_2/home_work2_6.py:6]: as I mentions in the lesson this HW should be done only using functions of types which were shown

File: lesson_2/home_work2_4.py (hunk 2 above)
Comment (SerhiiRiabko) [reply lesson_2/home_work2_4.py]: fixed

File: lesson_2/home_work2_6.py (hunk 4 above)
Comment (SerhiiRiabko) [reply lesson_2/home_work2_6.py:4]: I have changed - but I do not see changes now((

**Pull request closed without merging**